content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,14 +7,31 @@ menu:
     name: Release notes
 ---
 
+## v1.6.2 [2021-09-24]
+
+### Features
+
+- Add the `-template-id` property to the `GET /kapacitor/v1/tasks` request response. Adding this property helps to identify tasks that were created from a template.
```

Comment:
`s/-template-id/template-id` I think

Reply:
@lesam do you mean replace `-template-id` with `s/-template-id/template-id`?

Reply:
when you send the `GET /kapacitor/v1/tasks` requests, the `template-id` is included in the response

Reply:
Sorry, I meant replace `-template-id` with `template-id`.